layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md

```diff
@@ -0,0 +1,17 @@
+To send Datadog Agent logs to the Observability Pipelines Worker, update your Datadog helm chart [values.yaml][1021] with the following environment variables. See [Agent Environment Variables][1022] for more information.
```

Comment:
```suggestion
To send Datadog Agent logs to the Observability Pipelines Worker, update your Datadog Helm chart [`datadog-values.yaml`][1021] with the following environment variables. See [Agent Environment Variables][1022] for more information.
```

More recently, that file is referred to as `datadog-values.yaml` (previously we just called it `values.yaml`)

Reply:
Good to know!